frontend/src/language/no.json

```diff
@@ -170,13 +170,17 @@
     "Queue unscheduled missions": "Legg uplanlagte oppdrag i kø",
     "Queue all missions": "Legg alle oppdragene i kø",
     "Some missions are already in the queue": "Noen oppdrag står allerede i køen",
-    "Send robots to safe zone": "Send roboter til trygg sone",
-    "Send robots to safe zone long text": "Du er i ferd med å sende robotene til en trygg sone og dermed stoppe alle pågående oppdrag. For å kjøre oppdragene igjen må robotene starte fra begynnelsen.",
+    "Send robots to safe zone": "Send robotene til trygg sone",
+    "Send robots to safe zone long text": "Du er i ferd med å sende robotene til en sikker sone og dermed stoppe alle pågående oppdrag. Robotene vil fortsette på oppdragene når du bekrefter at det er trygt å fortsette.",
     "Send robots to safe confirmation text": "Er du sikker på at du vil sende alle robotene til en trygg sone?",
-    "Dismiss robots from safe zone long text": "Vanlig drift kan fortsette ved å starte oppdragene i køen eller ved å slette dem.",
-    "Dismiss robots from safe zone": "Slipp robotene ut av trygg sone",
+    "Dismiss robots from safe zone long text": "Du er i ferd med å fortsette oppdragene og dermed sende robotene ut av sikker sone.",
+    "Dismiss robots from safe confirmation text": "Er du sikker på at det er trygt å sende roboten ut av trygg sone?",
+    "Dismiss robots from safe zone": "Trygt å fortsette oppdrag",
     "Continue missions": "Fortsett oppdragene",
     "Safe Zone": "Trygg sone",
+    "Safe zone banner text": "Robotene kjører til sikker sone og vil ikke kjøre oppdrag. For å fortsette oppdrag trykk på knappen 'Trygt å fortsette oppdrag'.",
+    "Dismiss safe zone banner text": "Roboten fortsetter på oppdragene.",
```

Comment:
Plural on most but singular robot on 182

Suggest changing from "Roboten" to "Robotene"